src/mail.lisp

```diff
@@ -5,7 +5,6 @@
 
 
 (reblocks-auth/providers/email/resend:define-code-sender send-code ("Ultralisp.org <noreply@ultralisp.org>" url)
-  (:p ("To log into [Ultralisp.org](~A), follow [this link](~A)."
-       url
-       url))
+  (:p ("To log into Ultralisp https://ultralisp.org/, follow this link:"))
+  (:p ("~A" url))
```

Comment:
It is better to rewrite this as a proper `a` tag:

```
(:p (:a :href url
        url))
```

Reply:
Done.